resources/views/admin/users/word_filters.twig

```diff
@@ -0,0 +1,37 @@
+{% extends "layouts.admin" %}
+{% block title %}{% endblock %}
+{% block contents %}
+    <div class="admin-body__header">
+        <h1 class="admin-body__header__title">{{ trans('admin::word_filters.title') }}</h1>
+        <div class="admin-body__header__buttons">
+            <a href="{{ url_route('admin.word_filters.add') }}" class="button"><i class="fa fa-plus"></i> {{ trans('admin::word_filters.add_button') }}</a>
+        </div>
+    </div>
+    <div class="admin-body__content" style="display: flex;flex-direction: row;">
+        <table class="admin-table admin-table--bordered users" style="width:79%;">
+            <thead>
+            <tr>
+                <th>{{ trans('admin::word_filters.find_field') }}</th>
+                <th>{{ trans('admin::word_filters.replace_field') }}</th>
+                <th>{{ trans('admin::general.actions_field') }}</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for filter in word_filter_items %}
```

Comment:
Probably a good idea to use an [`else`](https://twig.symfony.com/doc/2.x/tags/for.html#the-else-clause) block to display a message if the collection is empty.